poc/vdaf.py

```diff
@@ -208,7 +208,7 @@ def run_vdaf(Vdaf,
             outbound.append(share)
 
         # Aggregators recover their output shares.
-        for i in range(Vdaf.ROUNDS):
+        for i in range(Vdaf.ROUNDS-1):
             # REMOVE ME
```

Comment:
I assume that the `# REMOVE ME` bits are purposefully not reflected in the final "post-loop" iteration on L227-233, but flagging this just in case.

Reply:
Great catch! This is a bug actually, we weren't producing the test vectors properly. Fixed and verified that the test vectors haven't changed.